feature/metis/manage-conversations/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/manageconversations/ui/conversation/create_personal_conversation/PotentialRecipientsUi.kt

```diff
@@ -135,10 +145,34 @@ private fun PotentialRecipientsList(
             }
         }
     } else {
-        Box(modifier = modifier) {
+        val icon = if (queryTooShort) Icons.Default.KeyboardAlt else Icons.Default.PersonOff
+        val text = if (queryTooShort) {
+            stringResource(
+                id = R.string.conversation_member_selection_query_too_short,
+                MemberSelectionBaseViewModel.MINIMUM_QUERY_LENGTH
+            )
+        } else {
+            stringResource(id = R.string.conversation_member_selection_recipients_empty)
+        }
+
+        Column(
+            modifier = modifier,
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.Center,
```

Comment:
Are you sure this is necessary? I think Arrangement.spacedBy(8.dp) would achieve the same thing without needing the spacer.